tests/fxl-orientation_duplicate/OPS/package.opf

```diff
@@ -0,0 +1,35 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<package xmlns="http://www.idpf.org/2007/opf" xmlns:epub="http://www.idpf.org/2007/ops" version="3.0" xml:lang="en" unique-identifier="pub-id">
+  <metadata xmlns:dc="http://purl.org/dc/elements/1.1/">
+    <dc:coverage>Fixed Layout</dc:coverage>
+    <dc:creator>Wendy Reid</dc:creator>
+    <dc:date>2021-09-28</dc:date>
+    <dc:description>EPUB creators must not declare the orienation property more than once.</dc:description>
+    <dc:identifier id="pub-id">fxl-orientation_duplicate</dc:identifier>
+    <dc:language>en</dc:language>
+    <dc:title>rendition:orientation duplicate</dc:title>
```

Comment:
Nit: "duplication", to match your other <title> elements. Noun is better in a title anyway.